### PR TITLE
Use ProcessBuilder and redirected log file for HDFS Extract

### PR DESCRIPTION
**Pull request by ericsun2**
schemaFetch.jar will generate log file along side with the output temp metadata file (which is defined in wh_etl_job_property under the name "hdfs.local.raw_metadata"). 

If hdfs.local.raw_metadata is set to /my_data/hdfs/abc.json, the log file will be /my_data/hdfs/abc.json.log

### CODE REVIEW

File: metadata-etl/src/main/java/metadata/etl/dataset/hdfs/HdfsMetadataEtl.java [hunk 1]
```diff
@@ -95,20 +98,23 @@ private void extractLocal()
...
-            + " -D " + Constant.HDFS_CLUSTER_KEY + "=" + cluster
-            + " -D " + Constant.HDFS_WHITE_LIST_KEY + "=" + whiteList
-            + " -D " + Constant.HDFS_NUM_OF_THREAD_KEY + "=" + numOfThread
-            + " -D " + Constant.HDFS_REMOTE_USER_KEY + "=" + hdfsUser
-            + " -D log_file_name=hdfs_schema_fetch" 
-            + " -D " + Constant.HDFS_REMOTE_KEYTAB_LOCATION_KEY + "=" + hdfsKeyTab;
-    //logger.info("executue remote command : " + execCmd);
-
-    Process process = Runtime.getRuntime().exec(execCmd);
+    String hdfsExtractLogFile = outputSchemaFile + ".log";
```
Comment (SunZhaonan) [inline metadata-etl/src/main/java/metadata/etl/dataset/hdfs/HdfsMetadataEtl.java]: This is good, separate the log for different job type makes it much easier for debugging, which were an headache before.

File: metadata-etl/src/main/resources/jython/HdfsLoad.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def load_metadata(self):
         update stg_dict_dataset
         set name = substring_index(urn, '/', -2)
         where db_id = {db_id}
-          and name in ('1.0', '2.0', '3.0', '4.0', '0.1', '0.2', '0.3', '0.4', 'dedup', '1-day', '7-day');
+          and name '[0-9]+\\.[0-9]+|dedup|dedupe|[0-9]+-day';
```
Comment (SunZhaonan) [inline metadata-etl/src/main/resources/jython/HdfsLoad.py]: Does this works? I think there should be a "regexp" between name and the pattern.

Comment (ericsun2) [inline metadata-etl/src/main/resources/jython/HdfsLoad.py]: "regexp" has been pasted back.

**Comment by jerrybai2009** [comment]
LGTM

**Pull request merged**